onecx-custom-auth/libs/my-custom-auth/src/lib/my-custom-auth-service.ts

```diff
@@ -0,0 +1,43 @@
+import { AuthService, AuthServiceFactory, Injectables } from './auth.service';
+
+export class MyCustomAuthService implements AuthService {
+  constructor(
+    private keycloakAuthService: AuthService,
+    config: Record<string, string>
+  ) {}
+  async init(config?: Record<string, unknown> | undefined): Promise<boolean> {
+    console.log('MyCustomAuthService: start init');
+    const value = await this.keycloakAuthService.init(config);
+    console.log('MyCustomAuthService: initResult', value);
+    return value;
+  }
+  getHeaderValues(): Record<string, string> {
+    const headerValues = this.keycloakAuthService.getHeaderValues();
+    console.log(
```

Comment:
getHeaderValues is called very often so I would not add logs here because it would span the console